ray-operator/config/samples/ray-job.kueue-toy-sample.yaml

```diff
@@ -26,14 +26,14 @@ spec:
 
   # rayClusterSpec specifies the RayCluster instance to be created by the RayJob controller.
   rayClusterSpec:
-    rayVersion: '2.9.0'
+    rayVersion: '2.41.0'
```

Comment:
This RayJob fails to succeed because of https://github.com/ray-project/kuberay/issues/2943. This is not related to Ray version. Merge this PR.